_includes/manuals/1.10/1.2_release_notes.md

```diff
@@ -2,6 +2,40 @@
 
 ### Headline features
 
+#### Orchestration rebuilder
```

Comment:
We use it in the manual introduction and the workflow section too.  It seems reasonable to use to describe the process to me.